Example/PaymentSheet Example/PaymentSheet Example/AppearancePlaygroundView.swift

```diff
@@ -97,8 +98,8 @@ struct AppearancePlaygroundView: View {
         )
 
         let selectedBorderWidthBinding = Binding(
-            get: { appearance.selectedBorderWidth ?? appearance.borderWidth * 1.5 },
-            set: { self.appearance.selectedBorderWidth = $0 }
+            get: { appearance.selectedBorderWidth ?? -0.5 },
```

Comment:
Maybe add a comment here documenting that -0.5 is a sentinel value used for `nil`